.github/workflows/ci.yml

```diff
@@ -491,6 +491,31 @@ jobs:
           ndk-version: ${{ matrix.ndk }}
           add-to-path: false
 
+      - name: Setup Java for Unity
+        if: ${{ matrix.platform == 'Android' }}
+        run: |
+          switch -Regex ("${{ matrix.unity-version }}")
+          {
+              "2019" {
+                  $javaHome = $env:JAVA_HOME_8_X64
+                  Write-Host "Using Java 8 for Unity 2019"
+              }
+              "2022" {
+                $javaHome = "$env:JAVA_HOME_11_X64"
```

Comment:
Do these need to be installed by us before hand or they are part of the build agent's image?
If something is missing (one of these paths is empty/undefined) how does the error manifest? Do we have some sane way to immediately stop saying "Didn't find X, we need that to work"